### PR TITLE
Release 2.0.2

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Release 2.0.2.
| Type?         | improvement

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+GitHub contributors:
+--------------------------------
+ - Francois Gaillard
+ - Jerome Nadaud
```
Comment (atomiix) [inline CONTRIBUTORS.md]:
```suggestion
```

**Review by matks — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Comment by florine2623** [comment]
Hello @PierreRambaud ,

The upgrade from 2.0.1 to 2.0.2 cannot be performed :
Screenshot 2022-01-04 at 14 55 55

Could you take a look ?
Thanks!

File: upgrade/upgrade-2.0.2.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+/**
+ * @param Module $module
+ *
+ * @return bool
+ */
+function upgrade_module_2_0_2$module)
```
Comment (PierreRambaud) [inline upgrade/upgrade-2.0.2.php]:
```suggestion
function upgrade_module_2_0_2($module)
```

**Comment by PierreRambaud** [comment]
> Hello @PierreRambaud ,
> 
> The upgrade from 2.0.1 to 2.0.2 cannot be performed : Screenshot 2022-01-04 at 14 55 55
> 
> Could you take a look ? Thanks!

Should be better right now!

**Comment by florine2623** [comment]
Thanks @PierreRambaud , LGTM now !
I added QA ✅ label !

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @PierreRambaud & @florine2623

**Pull request merged**